README.md

```diff
@@ -43,8 +49,9 @@ kubectl apply -k "github.com/ray-project/kuberay/manifests/base?ref=${KUBERAY_VE
 
 ### Use Helm
 
-A helm chart is a collection of files that describe a related set of Kubernetes resources. It can help users to deploy ray-operator and ray clusters conveniently.
-Please read [kuberay-operator](helm-chart/kuberay-operator/README.md) to deploy an operator and [ray-cluster](helm-chart/ray-cluster/README.md) to deploy a custom cluster.
+A Helm chart is a collection of files that describe a related set of Kubernetes resources.
+It can help users to deploy the KubeRay Operator and Ray clusters conveniently.
+Please read [kuberay-operator](helm-chart/kuberay-operator/README.md) to deploy the operator and [ray-cluster](helm-chart/ray-cluster/README.md) to deploy a configurable Ray cluster. To deploy the optional KubeRay API Server, see [kuberay-apiserver](helm-chart/kuberay-apiserver/README.md).
```

Comment:
`helm-chart/kuberay-operator/README.md` or `https://github.com/ray-project/kuberay/blob/master/helm-chart/kuberay-operator/README.md`? I am not a MarkDown expert and am not sure whether `helm-chart/kuberay-operator/README.md` will work or not.

Reply:
> I am not a MarkDown expert

Neither am I, but looks like it works :)
https://github.com/DmitriGekhtman/kuberay/tree/dmitri/update-readme#use-helm